.github/workflows/ci.yml

```diff
@@ -261,9 +266,44 @@ jobs:
               echo "found=$found" >> $GITHUB_OUTPUT
           fi
 
+      - name: Install libc++
```

Comment:
It's good to include some comments here because this is very confusing. I undertand why this is being done, but it's very confusing to anyone new at this.